modmail.py

```diff
@@ -258,6 +258,7 @@ def main(subreddit_name):
 
 			# Handle replying to the message with our private summary
 			# ...but don'r reply if the action was taken by USL Bot
+			# this is because it will send way too many notices about USLBot bans when most mods don't care
```

Comment:
Given the previous notes, this comment isn't needed here. If you want to give more reasoning below, please change the language away from "it will send way too many notices about USLBot bans..." This makes it seem like the bot is sending more than one notice per ban which is not the case. Instead, keep it objective and say something along the lines of "Auto-archiving messages reduces the noise created by USLBot in modmail."